.github/workflows/ci.yml

```
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    # branches:
+    #   - master
```

Probably want to uncomment before merging to avoid double builds
```suggestion
    branches:
      - master
```